### PR TITLE
Add 4S-EASST slides

### PR DESCRIPTION
**Pull request by Bisaloo**
- [x] The post specifies a license if you don't want to use the default CC BY
- [x] All authors have an ORCID iD
- [x] Relevant keywords / tags has been added. In particular, if you want your post to be shared on R-bloggers, you must tag it with `R`
- [x] Images or other external resources have been committed and pushed
- [x] The post uses [pure quarto syntax, rather than HTML or R code, unless necessary](../CONTRIBUTING.md#pure-quarto-syntax)

Right before merging:

- [x] The `date` field has been updated
- [x] All reviewers have been acknowledged in a short paragraph
- [ ] A PR has been opened in the [`blueprints`](https://github.com/epiverse-trace/blueprints) to link to this post
- [ ] The post has been re-rendered and content of the `_freeze/` folder is up-to-date

### CODE REVIEW

File: slides/2024-4S-EASST/index.qmd [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "The twin peaks of the global and the local: worldwide and modest infrastructures for global health"
+author:
+  - name: "Miller Díaz Valderrama"
+    orcid: "0000-0001-5583-0616"
+  - name: "Natalia Niño"
```
Comment (chartgerink) [inline slides/2024-4S-EASST/index.qmd:6]:
```suggestion
  - name: "Natalia Niño-Machado"
```

To be in line with the slides.

File: slides/2024-4S-EASST/index.qmd [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Based on a robust tradition of reflections in Science and Technology Studies, Critical Global Health, and Critical Data Studies about context (De Laet M., 2000; Tsing, A., 2005; Adams, 2016; Biehl 2016; Montgomery, C. M., Kingori, P., Sariola, S., & Engel, N, 2017; Lee, F 2021; 2023; Hepp, A; Jarke J; Kramp L 2022), we propose a symmetrical understanding of local-global relations in Global Health projects and enterprises. For this, we offer some examples in which we can see the operation of this kind of understanding, drawing from our involvement with two global projects on emergency preparedness, targeting the production of technologies for the analysis and control of infectious diseases.
```
Comment (chartgerink) [inline slides/2024-4S-EASST/index.qmd:20]: Do we want/need to include the exact references, as we can with a bib file?

**Review by chartgerink — APPROVED** [review]
Thanks! I left two suggestions but I don't think these are blockers at this time.

File: slides/2024-4S-EASST/index.qmd (hunk 2 above)
Comment (Bisaloo) [reply slides/2024-4S-EASST/index.qmd:20]: It would be nice in theory but probably not worth the extra effort

**Pull request merged**